app/javascript/bundles/Dashboard/components/SettingsComponents/ProfileSettings.jsx

```diff
@@ -26,7 +26,7 @@ const ProfileSettings = ({profile_pic_path}) => {
 		}
 	};
 	
-	const profilePic = profile_pic_path ? profile_pic_path : "/images/default_profile_pic.jpg"
+	const profilePic = profile_pic_path ? `data:image/jpeg;base64,${profile_pic_path}`: "/images/default_profile_pic.jpg";
```

Comment:
Another way to do this might be to always have profile_pic be set in the DB and have it be either default in the db or the specific S3 url, and then have that passed down from Db-> controller -> front end, removing this logic.